amgut/handlers/auth_handlers.py

```diff
@@ -42,7 +42,7 @@ def post(self):
             ag_login_id, skid, kitinfo['password'],
```

Comment:
I think that would work nicely, since it catches any issues with passwords not being migrated silently for the user.